static/js/dashboard_routes.js

```diff
@@ -0,0 +1,99 @@
+import App from './containers/App';
+
+const errorLoading = error => {
+  throw new Error(`Dynamic page loading failed: ${error}`);
+};
+
+const loadRoute = cb => (
+  module => cb(null, module.default)
+);
+
+const username = SETTINGS.user ? SETTINGS.user.username : '';
+
+export const routes = {
```

Comment:
Is it possible to have a function with these states returned as react components, then translate it via map to this dictionary? Or the other way around for the test routes?